src/main/promptToSaveWork.ts

```diff
@@ -17,17 +21,9 @@ const promptToSaveWork: (
 
   const fileName = path.basename(appState.getRepresentedFilePath() ?? '');
 
-  const buttonIndex = dialog.showMessageBoxSync(mainWindow, {
-    message:
-      fileName === ''
-        ? `Do you want to save your changes?`
-        : `Do you want to save the changes you made to ${fileName}?`,
-    detail: `Your changes will be lost if you don't save them.`,
-    buttons: ['Save', `Don't Save`, 'Cancel'],
-    type: 'warning',
-  });
-
-  if (buttonIndex === 0) {
+  const buttonIndex = saveChangesDialog(mainWindow, fileName);
+
+  if (buttonIndex === SaveDialogSelections.SAVE_SELECTED) {
     // Save, and tell the app to close after it's saved
     mainWindow.webContents.send('initiate-save-project', closeAfter);
```

Comment:
Now that you implemented the SaveDialogSelections enum, Could you update line 34 to use the the enum as well rather than checking if it equal to 1?

```javascript
  if (buttonIndex === 1) { // <-- this line, im not sure if github will show this on the review
    // Don't save
    return true;
  }
```

Reply:
Good pick up completely missed that one! Just fixed it